include/picongpu/plugins/particleCalorimeter/ParticleCalorimeter.x.cpp

```diff
@@ -301,8 +301,8 @@ namespace picongpu
             this->maxYaw_deg = float_X(0.5) * this->openingYaw_deg;
             this->maxPitch_deg = float_X(0.5) * this->openingPitch_deg;
             /* convert units */
-            const float_64 minEnergy_SI = this->minEnergy * sim.si.conv.ev2Joule(1.0e3);
-            const float_64 maxEnergy_SI = this->maxEnergy * sim.si.conv.ev2Joule(1.0e3);
+            const float_64 minEnergy_SI = this->minEnergy * sim.si.conv().eV2Joule(1.0e3);
+            const float_64 maxEnergy_SI = this->maxEnergy * sim.si.conv().eV2Joule(1.0e3);
```

Comment:
bug
```suggestion
            const float_64 minEnergy_SI = this->minEnergy * sim.unit.energy();
            const float_64 maxEnergy_SI = this->maxEnergy * sim.unit.energy();
```

Reply:
it conversion to keV, I will add this to the comment

Reply:
changed to 

```
            /* convert keV to joule */
            const float_64 minEnergy_SI = sim.si.conv().eV2Joule(this->minEnergy * 1.0e3);
            const float_64 maxEnergy_SI = sim.si.conv().eV2Joule(this->maxEnergy * 1.0e3);
            /* convert into PIConGPU units */
            this->minEnergy = minEnergy_SI / sim.unit.energy();
            this->maxEnergy = maxEnergy_SI / sim.unit.energy();
```